### ISSUE TITLE
LTR Loss issue

### ISSUE THREAD

**Issue by Aurora-LiZhi** [GRAPH-0/RGTN-NIE#5]
Hi , I am very interested in your paper and read it carefully .  But I don't understand the LRT Loss very well.I think if the predicted value closer to the ground truth, the loss value becomes smaller, isn't it? But for node v , if ground truth importance value is very small ,and predicted value is very big ,two values are very different .  After Softmax and LRT Loss function ，I think the  LRT Loss is a positive number approaching 0 , so small. Conversely, the loss value will be large. I can not understand the reason why the two values are very different but the loss value becomes so small, so I want to ask you this question

**Comment by GRAPH-0** [comment]
Thanks for your interest in this paper.
The LTR loss can be regarded as an auxiliary loss function of the RMSE loss. 
Ideally, the RMSE loss pushes the predicted value close to the ground truth, and the LTR loss is more concerned with the relative numerical size between the entity importance scores. In the experiment, there are related hyperparameters that control the weight of these two terms. In this way, we better model the relative relationship of importance between entities.

**Comment by Aurora-LiZhi** [comment]
> Thanks for your interest in this paper. The LTR loss can be regarded as an auxiliary loss function of the RMSE loss. Ideally, the RMSE loss pushes the predicted value close to the ground truth, and the LTR loss is more concerned with the relative numerical size between the entity importance scores. In the experiment, there are related hyperparameters that control the weight of these two terms. In this way, we better model the relative relationship of importance between entities.

Ok , I konw it. Thank you
